test/functional-k8s.bats

```diff
@@ -176,6 +176,11 @@ spec:
   done
 }
 
+@test "Prune Agent" {
+  local AGENT_NAME="${NAME}-0"
+  test iofogctl prune agent "$AGENT_NAME"
```

Comment:
This only tests if the command `iofogctl prune agent "$AGENT_NAME"` returned 0.
It does not test if the command worked properly.

You will need to find a way to check that the flag has been correctly set on Controller